tests/vmi_lifecycle_test.go

```diff
@@ -1050,7 +1050,7 @@ var _ = Describe("VMIlifecycle", func() {
 					data, err := logsQuery.DoRaw()
 					Expect(err).ToNot(HaveOccurred(), "Should get logs")
 					return string(data)
-				}, 30, 0.5).Should(ContainSubstring(fmt.Sprintf("Grace period expired, killing deleted VirtualMachineInstance %s", vmi.GetObjectMeta().GetName())), "Should log gracefull kill")
+				}, 30, 0.5).Should(ContainSubstring(fmt.Sprintf("Remove watchdog file /var/run/kubevirt/watchdog-files/kubevirt-test-default_%s", vmi.GetObjectMeta().GetName())), "Should log gracefull kill")
```

Comment:
That change is surprising.

Reply:
For me also, This test started failing once I finished working on the api. I double checked the flow of the graceful shutdown. 
VM really gets shutdown and the watchdog is the last step in that process. 

The killing can occur, but it is due to lack of communication from the VM. The more I think about it, the more I am convinced there have to be negative test for this case. Will create an issue for it.